docs/tracking-plan.md

```diff
@@ -0,0 +1,63 @@
+# Full stack (data) submission documentation
+
+## Modification to the app
+
+The feature that I've implemented is a simple cart. By clicking the Buy button on the products listed below.
+
+![Add to cart](add-to-cart.gif)
+
+This change assumed a logged in user and valid data being passed from React to the Apollo server. Additionally, the implementation is such that the "database" is stored in memory inside the `cart` array which is then returned by `Query cart` and is then mutated by `Mutation addToCart`. In a proper implementation this data would be stored in the database but this is too tedious to implement in this case.
+
+We have also added a "Checkout" button to the cart. This button doesn't do anything but is connected to our GTM container tracking. See the Tracking Plan below for more details.
```

Comment:
Clear documentation thank you. I have reviewed it fully. We will be contact.

Reply:
Sounds great - looking forward to hearing from you and Bonsai. Have a great evening!